partuniverse/templates/pmgmt/part/detail.html

```diff
@@ -93,7 +93,8 @@
                         <a href="{% url "storage_item_detail" item.id %}">{{item.storage}}</a>
                     </div>
                 {% if item.on_stock %}
-                    ({{item.on_stock}}{{object.unit}})<a href="{% url "storage_item_transaction" item.id %}" class="ui button mini teal right floated compact">{% trans "Update stock" %}</a>
+                    ({{item.on_stock}}{{object.unit}})<a href="{% url "storage_item_transaction" item.id %}" class="ui button mini teal right floated compact">{% trans "Update stock" %}</a> <a href="{% url "storage_item_update" item.id %}" class="ui button mini teal right floated compact">{% trans "Edit stock" %}</a>
```

Comment:
Maybe tweaking the wording a little? 
StorageItemUpdateView is not including the actual stock

Reply:
So, "Edit storageitem" should be correct?